libs/sdk-ui-pivot/src/impl/resizing/columnSizing.ts

```diff
@@ -760,13 +760,33 @@ function collectWidths(
     font: string,
     rowIndex: number,
 ): void {
-    const { context } = config;
+    const { context, gridApi, columnApi } = config;
 
     const col = config.tableDescriptor.getCol(column);
+    const colDef = column.getColDef();
 
     if (col && context) {
         const text = row[col.id];
-        const formattedText = isSeriesCol(col) && valueFormatter(text, col, config.separators);
+        let formattedText = undefined;
+        const valueFormatter = colDef.valueFormatter;
+        if (valueFormatter) {
```

Comment:
Possibly this would look bit better if written as:

if (valueFormatter  && shouldApplyFormatting(col, config) && typeof valueFormatter === "function") {
   formattedText = valueFormatter({data: row, ... });
}

* `shouldApplyFormatting` can receive config and isTransposed can be checked inside of it
* Either use invariant or typeof condition. I don't see necessity to use both. If you want invariant, you can hide check in function so it can be called in if statement. Possibly whole if condition can be constant.